Include/internal/pycore_tuple.h

```diff
@@ -8,6 +8,7 @@ extern "C" {
 #  error "this header requires Py_BUILD_CORE define"
 #endif
 
+#include "pycore_object.h"            // _PyObject_GC_IS_TRACKED
```

Comment:
```suggestion
#include "pycore_object.h"        // _PyObject_GC_IS_TRACKED
```

I hope the `//` is aligned with the `_PyStackRef` as well.

Reply:
Yeah, it's not aligned, but I corrected it.